src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -22,9 +22,8 @@ public UserController(UserService userService,
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto findByEmail(@RequestParam String email) {
-        User user = userService.findByEmail(email).orElseThrow(
-                () -> new RuntimeException("Invalid email"));
+    public UserResponseDto findByEmail(Authentication authentication) {
```

Comment:
now our search can search only us, let's replace it with what we had before and add `@Email` to it